C#/C#.sublime-syntax

```diff
@@ -1,278 +1,1634 @@
 %YAML 1.2
 ---
 # http://www.sublimetext.com/docs/3/syntax.html
-name: "C#"
+# https://github.com/gwenzek/csharpSyntax
```

Comment:
Thanks, do you think a link to the MIT license is enough or shall I put it entirely ?
Also shall we remove the "C#/Readme.txt" since the information it contains is not accurate anymore ?